apps/server/src/modules/oauth/service/oauth.service.ts

```diff
@@ -123,21 +130,31 @@ export class OAuthService {
 
 	async processOAuth(query: AuthorizationParams, systemId: string): Promise<OAuthResponse> {
 		try {
+			this.logger.debug('Oauth process strated. Next up: checkAuthorizationCode().');
```

Comment:
@Metauriel debug level is okay here? What level is set for logs in staging and prod? i guess info?

Reply:
its definetly not debug :P
I think this is fine here.